libexec/bats-core/bats-exec-test

```diff
@@ -61,7 +66,7 @@ run() {
   local origIFS="$IFS"
   output="$("$@" 2>&1)"
   status="$?"
-  IFS=$'\n' lines=($output)
+  IFS=$'\n' "$output"
```

Comment:
https://www.shellcheck.net/wiki/SC2034

Not clear what is the purpose of `$lines`. Removed, but leaving the output